Translation/Extractor/File/FormExtractor.php

```diff
@@ -138,7 +142,7 @@ public function enterNode(Node $node)
                         $this->parseItem($item, $domain);
                         break;
                     case 'choices':
-                        if ($this->parseChoiceNode($item, $node, $domain)) {
+                        if ($this->parseChoiceNode($item, $node, $choiceDomain)) {
                             continue 2;
```

Comment:
Putting it in the `getDomain` is not an option as that is a separate domain that can be used for label (label and choices may have different domains). And I just followed the pattern of passing the domain to `parseChoiceNode` method.